UK/Area/London AC/Bandbox (TopSky).prf

```diff
@@ -3,12 +3,16 @@ Settings	SettingsfilePROFILE	\..\..\Data\Settings\Local\London\AC_BB_Profiles.tx
 Settings	SettingsfileSCREEN	\..\..\Data\Settings\Screen.txt
 Settings	SettingsfileVOICE	\..\..\Data\Settings\Voice.txt
 Settings	Settingsfile	\..\..\Data\Settings\AC\AC_General.txt
-Settings	SettingsfileARR	\..\..\Data\Settings\Lists_TopSky.txt
-Settings	SettingsfileCONFLICT	\..\..\Data\Settings\Lists_TopSky.txt
-Settings	SettingsfileDEP	\..\..\Data\Settings\Lists_TopSky.txt
-Settings	SettingsfileFP	\..\..\Data\Settings\Lists_TopSky.txt
-Settings	SettingsfileSEL	\..\..\Data\Settings\Lists_TopSky.txt
-Settings	SettingsfileSIL	\..\..\Data\Settings\Lists_TopSky.txt
+Settings	SettingsfileARR	\..\Data\Settings\Lists_TopSky.txt
```

Comment:
@RedstonePilot  There is a difference between the spaces used here.

I believe ES needs a tab, rather than a space for these lines to work.

Reply:
Interesting it was a copy and paste and I just edited the path at the end. Guess something messed up. Will look when free.

Reply:
@luke11brown Its showing as a tab for me? 
![image](https://github.com/VATSIM-UK/uk-controller-pack/assets/146777065/4a2317b3-681d-489a-99f3-c0b6db783c64)
Not sure what's happened

Reply:
Why do some have \..\..\Data while some have only a single \..\?
Regarding the tab issues they look like tabs my end